docs/sources/tempo/configuration/grafana-alloy/tail-sampling/policies-strategies.md

```diff
@@ -0,0 +1,585 @@
+Tail sampling operates as part of the telemetry processing pipeline in Alloy: it can make informed decisions based on the entire trace as opposed to isolated spans.
+The sampling criteria policies are defined within a tail sampling block.
+For more information, refer to [tail sampling processor block](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/).
+
+For example, this basic Alloy configuration receives OTLP data either via HTTP or gRPC and passes incoming trace spans into the tail sampling processor.
+Before sending the trace spans to a tracing store like Tempo, the tail sampling processor decides whether or not to sample the trace based on the probabilistic policy.
+The probabilistic policy simply samples a specified percentage of traces observed randomly.
+Defaults are used for the tail sampling processor.
+
+```alloy
+// Expose receiving OTLP data
+otelcol.receiver.otlp "example" {
+  // Allow OTLP HTTP data on all interfaces on port 4318
+  http {
+    endpoint = "0.0.0.0:4318"
+  }
+  // Allow OTLP gRPC data on all interfaces on port 4317
+  grpc {
+    endpoint = "0.0.0.0:4317"
+  }
+
+  // Send all received trace spans to the tail sampling processor
+  output {
+    traces = [ otelcol.processor.tail_sampling.example.input ]
+  }
+}
+
+// Make decisions on whether to sample or discard traces
+otelcol.processor.tail_sampling "example" {
+  // We define a single probabilistic processor to determine sampling
+  policy {
+    // The name of the policy, each policy name must be unique for this tail sampling instance
+    name = "example_probabilistic"
+    // The policy type is probabilistic
+    type = "probabilistic"
+
+    // Each policy type is defined by a block for the policy with specific parameters for it
+    probabilistic {
+      // The overall ratio of traces that have been received to randomly sample.
+      // In this case 1 in 10.
+      sampling_percentage = 10
+    }
+  }
+
+  // Output all sampled trace spans to the OTLP exporter.
+  output {
+    traces = [ otelcol.exporter.otlp.example.input ]
+  }
+}
+
+// The OTLP exporter sends telemetry onwards to a downstream destination.
+otelcol.exporter.otlp "example" {
+  // The client block defines the target destination.
+  client {
+    // The endpoint denotes the location of the host receiving the sampled trace
+    // data. In this case a local Tempo instance.
+    endpoint = "http://tempo:4318"
+  }
+}
+```
+
+## Policy types
+
+The table lists available policy types for the tail sampling processor.
+For additional information, refer to [`otelcol.processor.tail_sampling` component](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/)` in the Alloy documentation and the [Tail Sampling Processor README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md) for the OTel Collector.
```

Comment:
The second ` is showing up when I look at preview mode. Probably need "the" beforehand too.

Reply:
```suggestion
For additional information, refer to the [`otelcol.processor.tail_sampling` component](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/otelcol.processor.tail_sampling/)` in the Alloy documentation and the [Tail Sampling Processor README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md) for the OTel Collector.
```